general_rules/ContinueRules.tex

```diff
@@ -111,7 +111,7 @@ \subsubsection{Alternative interfaces for HRI}
 Consider localization (with English as the default), but also potential users of service robots at their home.
 For example: elderly people and people with physical disabilities.
 
-\textbf{\textsc{Award:}} The best alternative is awarded the Best Human-Robot Interface award (\refsec{award:hri}).
+%\textbf{\textsc{Award:}} The best alternative is awarded the Best Human-Robot Interface award (\refsec{award:hri}).
```

Comment:
Are we dropping this award? Then just remove it.

Reply:
The award is described in the orga doc